cassandra/cqlengine/management.py

```diff
@@ -135,6 +135,20 @@ def sync_table(model):
 
     *There are plans to guard schema-modifying functions with an environment-driven conditional.*
     """
+
+    if keyspaces:
+        if not isinstance(keyspaces, (list, tuple)):
+            raise ValueError('keyspaces must be a list or a tuple.')
+
+        for keyspace in keyspaces:
```

Comment:
Do you think it's worth also including `Model.__keyspace__`, if set? Otherwise we might want to explain that the `keyspaces` parameter overrides model attribute and default keyspace.